DigitalOffice/Services/UserService/Controllers/UserController.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using UserService.Commands.Interfaces;
+using UserService.Models;
+
+namespace UserService.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        [Route("getUserById")]
+        [HttpGet]
+        public User GetUserById([FromServices] IGetUserByIdCommand getUserInfoByIdCommand, [FromQuery] Guid userId)
+            => getUserInfoByIdCommand.Execute(userId) ?? throw new Exception("User with this id not found.");
```

Comment:
For what exception? Check command class

Reply:
I wrote it in case the implementation of the IGetUserByIdCommand interface may not throw an error if there is no user in the database. Is this bad practice?